### ISSUE TITLE
Privacy information

### ISSUE THREAD

**Issue by juanvalag2019** [arcinxe/ArktinMonitor#4]
Hi, this is a unique and great project. I would like to know about the use of the data that is stored through Windows and Web app.

**Comment by rickb06** [comment]
Agreed. This project has some genuine promise of becoming something more, something unique and far more powerful than the paltry options available at present. I would **love** to see a self hosted version though, a general no-warranty .NET API would be great!
